charts/deepgram-self-hosted/values.yaml

```diff
@@ -112,7 +112,7 @@ api:
     # -- pullPolicy configures how the Kubelet attempts to pull the Deepgram API image
     pullPolicy: IfNotPresent
     # -- tag defines which Deepgram release to use for API containers
-    tag: release-240927
```

Comment:
Let's make sure to add `speak_streaming` to the `api.config.yaml` file, with a default of `true`, for this Helm chart, in addition to the Docker/Podman resources.

Reply:
Added, and removed the corresponding instruction in changelog, since it will not be necessary: https://github.com/deepgram/self-hosted-resources/pull/47/commits/864319f19e8a1df432be96c1ccb785dbc83e9f81